server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -190,6 +194,10 @@ public CompletableFuture<PushResultDto> push(
             CommitMessageDto commitMessage,
             @RequestConverter(ChangesRequestConverter.class) Iterable<Change<?>> changes) {
         checkPush(repository.name(), changes);
+        meterRegistry.counter("commits.push",
```

Comment:
Note: Didn't add this counter to ThriftService because it's going to be deprecated.